lms/djangoapps/teams/views.py

```diff
@@ -620,22 +623,37 @@ def get(self, request):
                 'user_message': _(u"The ordering {ordering} is not supported").format(ordering=ordering),
             }, status=status.HTTP_400_BAD_REQUEST)
 
-        page = self.paginate_queryset(topics)
-        serializer = self.pagination_serializer_class(page, context={'course_id': course_id, 'sort_order': ordering})
+        # Always sort alphabetically, as it will be used as secondary sort
+        # in the case of "team_count".
+        topics = get_alphabetical_topics(course_module)
+        if ordering == 'team_count':
```

Comment:
I actually find it quite readable as-is, so I'm not included to split it up. Thanks for the suggestion though-- something to keep in mind in general.